### PR TITLE
Small tomato killer tweak

### PR DESCRIPTION
**Pull request by TheShuEd**
## About the PR
add tomato killers in flora anomaly bulb spawns
damage from 4 brute -> 2 blunt + 2 piercing
Huge size -> Normal size

**Changelog**
:cl:
- add: Added tomato killers in floral anomaly berries
- tweak: tweak size and damage of killer tomatoes

### CODE REVIEW

**Comment by SoulFN** [comment]
I think it should just be made so that you can get no more than one tomato per harvest

**Comment by TheShuEd** [comment]
The division into different types of damage is due more to logic - tomatoes do bite, not hit enemies with blunt damage. It is also easier to heal divided damage on the contrary.

**Comment by TheShuEd** [comment]
size reduction is more of a QoL change. Almost no one carries them in their inventory anyway, they just pack them in the trash.

**Comment by TheShuEd** [comment]
no, its size in inventory. 4x4 before, 2x2 now

**Comment by metalgearsloth** [comment]
Don't think of damage types in "well flat weapons do blunt"; piercing is made for bullet damage types.

File: Resources/Prototypes/Entities/Mobs/NPCs/miscellaneous.yml [hunk 1]
```diff
@@ -136,16 +135,13 @@
   - type: MeleeWeapon
     hidden: true
     damage:
-      groups:
-        Brute: 4
+      types:
+        Blunt: 2
+        Piercing: 2
```
Comment (metalgearsloth) [inline Resources/Prototypes/Entities/Mobs/NPCs/miscellaneous.yml:140]: What's wrong with slashing.

**Comment by TheShuEd** [comment]
Revert damage changes

**Pull request merged**